<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="maximumSize">
   <size>
    <width>600</width>
    <height>500</height>
   </size>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="acceptDrops">
   <bool>false</bool>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLineEdit" name="Edit_URL">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>80</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_URL">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>87</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>URL:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_Login">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>110</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Login">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>117</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Login:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Password">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>148</y>
      <width>59</width>
      <height>15</height>
     </rect>
    </property>
    <property name="text">
     <string>Password:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="Edit_Password">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>140</y>
      <width>113</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_MainString">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>40</y>
      <width>321</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Enter the URL, login, and password of your repository:</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Connect">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>180</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Connect</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_BranchName">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>250</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Label_Select">
    <property name="geometry">
     <rect>
      <x>174</x>
      <y>220</y>
      <width>271</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select the branch which you want to follow:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Label_CheckNewCommits">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>280</y>
      <width>281</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Select interval for checking the new commits:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="ComboBox_TimeInterval">
    <property name="geometry">
     <rect>
      <x>229</x>
      <y>301</y>
      <width>151</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Add_Save">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>380</y>
      <width>91</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Add and Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="Button_Cancel">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>380</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Cancel</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>600</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
